reporter.go

```diff
@@ -234,7 +235,7 @@ func (r *remoteReporter) processQueue() {
 				atomic.AddInt64(&r.queueLength, -1)
 				if flushed, err := r.sender.Append(span); err != nil {
 					r.metrics.ReporterFailure.Inc(int64(flushed))
-					r.logger.Error(err.Error())
+					r.logger.Error(fmt.Sprintf("span %q: %s", err.Error()))
```

Comment:
Would something like: "Failed to report span ..." be more apt?

Reply:
Is this sufficient information? Does it make sense to report the whole span at a lower log level?